closed/custom/modules/java.base/Copy.gmk

```diff
@@ -189,21 +189,21 @@ endif # OPENJ9_ENABLE_DDR
 # To bundle first search for openssl 1.1.x library, if not found, search for 1.0.x
 ifneq ($(OPENSSL_BUNDLE_LIB_PATH), )
   ifeq (linux,$(OPENJDK_TARGET_OS))
-    LIBCRYPTO_NAMES := libcrypto.so.1.1 libcrypto.so.1.0.0
+    LIBCRYPTO_NAMES := libcrypto.so.3 libcrypto.so.1.1 libcrypto.so.1.0.0
```

Comment:
`LIBSSL_NAMES` also needs to include `libssl.so.3`.

Reply:
Updated.